packages/round-manager/src/features/program/CreateProgramPage.tsx

```diff
@@ -173,6 +206,22 @@ export default function CreateProgram() {
                       </p>
                     )}
                   </div>
+
+                  <div className="sm:basis-1/2">
+                    <label htmlFor="program-chain" className="block text-xs">
+                      <span className="opacity-50">
+                        Program Chain
+                      </span>
+                      <ProgramChain />
+                    </label>
+                    <Input
+                      type="text"
+                      disabled={true}
+                      data-testid="selected-chain"
+                      className="disabled:opacity-50"
+                      value={chain.name}
```

Comment:
Do we need a util function which given chain name returns logo ?  Currently the icon from rainbow and I'm not sure how to get that